core/src/debug/assets/undefined-mobile-services.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Probably pushed by mistake. File name is wrong etc.

Reply:
Oh no this is right.  It is for some of the new tests